tools/deployment/chocolatey/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,87 @@
+function Set-DenyWriteAcl {
```

Comment:
nit, potentially significant code reuse, is it possible to combine into a library or import?